backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/bpartner/process/C_BPartner_AddRemoveCreditStopStatus.java

```diff
@@ -31,8 +30,8 @@
 	private  final IBPartnerStatsDAO bpartnerStatsDAO = Services.get(IBPartnerStatsDAO.class);
 	private final IBPartnerDAO bpartnerDAO = Services.get(IBPartnerDAO.class);
 
-	@Param(parameterName = "IsSetCreditStop", mandatory = true)
-	private boolean isSetCreditStop;
+	@Param(parameterName = "setCreditStatus", mandatory = true)
+	private String setCreditStatus;
```

Comment:
consider using SetCreditStatusEnum instead of String.

will be handled by API because `ReferenceListAwareEnum` implementations are.

Reply:
change done